.github/workflows/javascript-typescript.yml

```diff
@@ -66,6 +66,15 @@ jobs:
 
         cd -
 
+    - name: Publish task
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      run: |
+        git config --global user.name "Github Actions"
+        git config --global user.email "matt.mccormick@kitware.com"
```

Comment:
Is there an itk robot account?

Reply:
`Github` -> `GitHub`

We can use `itk+community@discourse.itk.org` for the email address.